src/app/shared/top-panel/workspace-panel/resource-menu/pipes/can-create-resource.pipe.ts

```diff
@@ -31,7 +31,7 @@ import {Project} from '../../../../../core/store/projects/project';
 import {selectCurrentUser} from '../../../../../core/store/users/users.state';
 import {PermissionsPipe} from '../../../../pipes/permissions/permissions.pipe';
 
-const allowedEmails = ['support@lumeer.io', 'mvecera@lumeer.io', 'kubedo8@gmail.com', 'livoratom@gmail.com'];
+export const allowedEmails = ['support@lumeer.io', 'mvecera@lumeer.io', 'kubedo8@gmail.com', 'livoratom@gmail.com'];
```

Comment:
This should be moved to a new file and renamed to something more descriptive (e.g. `superUserEmails`) as we are giving it a global scope now.